src/style.css

```
@@ -0,0 +1,123 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-style: roboto, sans-serif;
+}
```

Цей шрифт не був підключений до сторінки
```suggestion
html {
  font-style: "Roboto", sans-serif;
}
```
Назву шрифта потрібно брати в ""